app/cdap/components/ConnectionList/Components/TabLabelCanBrowse/styles.ts

```diff
@@ -46,4 +50,7 @@ export const useStyles = makeStyles({
     color: '#fff',
     fontSize: 'large',
   },
+  connectorTypeIcon: {
+    paddingRight: '11px',
+  },
```

Comment:
use number directly instead of number with px

Reply:
resolved